notification/README.md

```diff
@@ -0,0 +1,254 @@
+--- 
+layout: pattern 
+title: com.iluwater.notification.Notification
+folder: notification 
+permalink: /patterns/notification/ 
+categories: Behavioural
+language: en 
+tags:
+- Decoupling
+- Presentation
+---
```

Comment:
Please check the updated yaml frontmatter requirements in https://github.com/iluwatar/java-design-patterns/wiki/01.-How-to-contribute.

To the tags we could also add `Domain`